### PR TITLE
Add runtime mark on title

### PR DESCRIPTION
**Pull request by initdc**

### CODE REVIEW

File: assembly/index.ts [hunk 1]
```diff
@@ -1,3 +1,33 @@
+const welcomePage = `<!DOCTYPE html>
```
Comment (Integralist) [inline assembly/index.ts:1]: We should keep this consistent (if possible, cc @torch2424) with the JS version https://github.com/fastly/compute-starter-kit-javascript-default/blob/main/src/index.js where we import the content from disk.

File: fastly.toml [hunk 2]
```diff
@@ -3,3 +3,4 @@ name = "Default starter for AssemblyScript"
 description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 authors = ["<oss@fastly.com>"]
 language = "assemblyscript"
+# service_id = ""
```
Comment (Integralist) [inline fastly.toml:6]: We should just remove the `service_id` altogether.

File: assembly/index.ts [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Welcome to AS on Compute@Edge</title>
```
Comment (Integralist) [inline assembly/index.ts:10]: The title across the other starter kits are also consistent...

- https://github.com/fastly/compute-starter-kit-javascript-default/blob/main/src/welcome-to-compute%40edge.html#L10
- https://github.com/fastly/compute-starter-kit-rust-default/blob/main/src/welcome-to-compute%40edge.html#L10

Comment (Integralist) [reply assembly/index.ts:10]: UPDATE: I've just seen there are identical PRs added to the other language starter kits. I personally see no value in adding the runtime to the title of this synthetic response. @triblondon thoughts?

Comment (kailan) [reply assembly/index.ts:10]: Oh, I just reviewed the Rust one. I assumed the PR was a mistake until I saw these.

Agreed that this change is redundant.

**Comment by Integralist** [comment]
Closing on the basis that the other PRs mentioned have also been closed now.

**Pull request closed without merging**